af/fastpath/fastpath/db.py

```diff
@@ -182,24 +184,25 @@ def tf(v: bool) -> str:
         msm["measurement_start_time"], "%Y-%m-%d %H:%M:%S"
     )
     test_start_time = datetime.strptime(msm["test_start_time"], "%Y-%m-%d %H:%M:%S")
-    row = [
-        measurement_uid,
-        nn(msm, "report_id"),
-        input_,
-        nn(msm, "probe_cc"),
-        asn,
-        test_name,
-        test_start_time,
-        measurement_start_time,
-        ujson.dumps(scores),
-        nn(msm, "platform"),
-        tf(anomaly),
-        tf(confirmed),
-        tf(msm_failure),
-        domain,
-        nn(msm, "software_name"),
-        nn(msm, "software_version"),
-    ]
+    row = dict(
+        measurement_uid=measurement_uid,
+        report_id=nn(msm, "report_id"),
+        input=input_,
+        probe_cc=nn(msm, "probe_cc"),
+        probe_asn=asn,
+        test_name=test_name,
+        test_start_time=test_start_time,
+        measurement_start_time=measurement_start_time,
+        scores=ujson.dumps(scores),
+        platform=nn(msm, "platform"),
+        anomaly=tf(anomaly),
+        confirmed=tf(confirmed),
+        msm_failure=tf(msm_failure),
+        blocking_type=nn(msm, "blocking"),
```

Comment:
Shouldn't this be extracted from the `analysis` field of the `scores`? Measurements don't have as a top level key the value `blocking`.

Reply:
No, it's from test_keys -> blocking and it's directly from the measurement so that it will support any compatible test in future

Reply:
I don't think we should be moving in the direction of putting more analysis logic inside of the probe and relying on them to generate a valid blocking key.

It should be the responsibility of the analysis engine to come up with a suitable blocking value based on the results of the measurement (only the backend is aware of the blocking fingerprints and we can improve the blocking heuristics iteratively).